trio/_tools/gen_exports.py

```diff
@@ -16,14 +16,22 @@
 from textwrap import indent
 from typing import TYPE_CHECKING
 
+import attr
+
 if TYPE_CHECKING:
     from typing_extensions import TypeGuard
 
+# keep these imports up to date with conditional imports in test_gen_exports
+# isort: off
```

Comment:
Instead of turning isort off/on, you can use `isort: split` to ensure imports don't get intermixed across the comment. Then it'll still sort the external tool imports.